test/test_integration_credentials_netrc.py

```diff
@@ -131,7 +130,7 @@ def test_no_netrc_and_logout_returns_successful_logout_message(self,mock_accessi
         '''
 
     @integration_test(True)
-    def test_cli_configured_in_insecure_mode_but_run_in_secure_mode_raises_error(self,keystore_disable_mock):
+    def test_cli_configured_in_insecure_mode_but_run_in_secure_mode_raises_error_netrc(self, keystore_disable_mock):
```

Comment:
for the mocks passed in we have both `keystore_disable_mock` and `mock_accessible`. I believe those are the same? If so, let's pick a name and use that. `keystore_disable_mock` is 👌  with me

Reply:
Also note that we have a mix of terminologies here `keystore` `keyring`. I also suggest we pick one